core/ir/aarch64/codec.py

```diff
@@ -561,9 +561,9 @@ def read_codec_file(path):
                     # Syntax: pattern opcode opndtype* : opndtype*
```

Comment:
> Issue: #2626 - escape sequences in codec.py rejected by Python 3.12

This does not follow the commit title conventions.  Please change it in the PR UI; that is what becomes the merged commit title.  It should be something like `i#2626 a64 decoder: escape sequences in codec.py rejected by Python 3.12`.  Please see https://dynamorio.org/page_code_reviews.html#sec_commit_messages